ui/src/app/settings/selectors/configuration.js

```diff
@@ -0,0 +1,26 @@
+const configuration = {};
+
+/**
+ * Returns all configuration.
+ * @param {Object} state - The redux state.
+ * @returns {Array} A list of all configuration.
+ */
+configuration.all = state => {
+  return state.configuration.items;
+};
```

Comment:
You can just say `configuration.all = state => (state.configuration.items)`.

Although `all` is not likely to be consumed directly by a component (see earlier comment), you can compose new, more specific selectors on top of this with reselect.